modules/serverless-autoscaling-minscale.adoc

```diff
@@ -0,0 +1,31 @@
+[id="serverless-autoscaling-minscale_{context}"]
+= Configuring the minScale annotation
+
+If the `minScale` annotation is not set, pods scale to zero, or to 1 if `enable-scale-to-zero` is set to `false` in the `config-autoscaler` config map.
+
+.Example service spec with `minScale` configured
```

Comment:
For consistency, would you mind formatting _Configuring the maxScale annotation_ and _Configuring the maxScale annotation_ as procedures?